modules/_UXTeam.scss

```diff
@@ -0,0 +1,113 @@
+@font-face {
+    font-family: Montserrat;
+    src: url(fonts/static/Outfit-ExtraLight.ttf);
+}
+#UXTeam{
+    // padding: 0 var(--size-8);
+    background-color: red;
+    text-align: center;
+
+.tittle{
+        padding: 2rem 0;
+        color:#000;
+        font-size: var(--size-6);
+        font-weight: 700;
+        line-height: 1.15;
+        .detail{
+            color:var(--orange-color);
+        }
+}
+p{
```

Comment:
Which element is this one? Be consistent in the selectors and try to avoid using elements so early in your selectors